### PR TITLE
Schema diagram

### PR DESCRIPTION
**Pull request by OnlyUnknown**
- We have added the diagram to the drawsql website
- We have added the file of schema_based_on_diagram file
- We have added the data that is required 
- Have added indexes
- Have added a new table to join the table between "medical_histore and Treatments"

### CODE REVIEW

File: schema_based_on_diagram.sql [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+    "Treatments" ADD PRIMARY KEY("id");
+CREATE TABLE "paitens"(
+    "id" INTEGER NOT NULL,
+    "name" VARCHAR(255) NOT NULL,
+    "date_of_birth" DATE NOT NULL
+);
+ALTER TABLE
+    "paitens" ADD PRIMARY KEY("id");
+ALTER TABLE
+    "Treatments" ADD CONSTRAINT "treatments_id_foreign" FOREIGN KEY("id") REFERENCES "medical_history"("id");
```
Comment (Gambit142) [inline schema_based_on_diagram.sql]: - [ ] The `medical_history` and the `treatments` tables have a **_many-to-many_** relationship based on the schema diagram provided. I would advise that you create a join table called `medical_histories_has_treatments` and create the foreign constraints as you would in a join table that connects other tables that have a _**many-to-many**_ relationship. 👍 💻 

#### Hint: Remove this constraint that you have created here. 👍

File: schema_based_on_diagram.sql [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+ALTER TABLE
+    "invoice_items" ADD CONSTRAINT "invoice_items_treatment_id_foreign" FOREIGN KEY("treatment_id") REFERENCES "Treatments"("id");
+ALTER TABLE
+    "invoices" ADD CONSTRAINT "invoices_medical_history_id_foreign" FOREIGN KEY("medical_history_id") REFERENCES "medical_history"("id");
+ALTER TABLE
+    "invoice_items" ADD CONSTRAINT "invoice_items_invoice_id_foreign" FOREIGN KEY("invoice_id") REFERENCES "invoices"("id");
+ALTER TABLE
+    "medical_history" ADD CONSTRAINT "medical_history_patient_id_foreign" FOREIGN KEY("patient_id") REFERENCES "paitens"("id");
+ALTER TABLE
+    "medical_history" ADD CONSTRAINT "medical_history_id_foreign" FOREIGN KEY("id") REFERENCES "Treatments"("id");
```
Comment (Gambit142) [inline schema_based_on_diagram.sql]: - [ ] The `medical_history` and the `treatments` tables have a **_many-to-many_** relationship based on the schema diagram provided. I would advise that you create a join table called `medical_histories_has_treatments` and create the foreign keys as you would in a join table that connects other tables that have a _**many-to-many**_ relationship. 👍 💻 

#### Hint: Remove this constraint that you have created here and add new ones to the new join table. 👍

Comment (Gambit142) [inline schema_based_on_diagram.sql]: - [ ] Kindly create indexes on the foreign keys you created in this schema file including the one that you will create on the join table. Creating indexes on foreign keys makes queries more efficient especially if you are querying a table that has many data entries. 👍

**Review by Gambit142 — CHANGES_REQUESTED** [review]
Hi @OnlyUnknown @KAYEMBAJONATHAN 👋, 

Good job so far!

## Highlights
- Tables were created based on schema diagram ✔️ 
- Every table has a primary key ✔️ 
- All tables were created in a new schema file ✔️ 

You still need to work on some issues to go to the next project, but you are almost there!

## Required Changes ♻️
_Kindly check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend considering them as they can improve your code._

You can also consider the following:

N/A

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### Please tag me in your question so I can receive the notification. You can tag me by typing @Gambit142

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Review by abdulhamiid — APPROVED** [review]
# :green_circle:  Approved! :partying_face:
Hi Team 👋, 
image
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 🌟

- ✔️ Interpret a database diagram and implement the SQL needed to build that database.

## Optional suggestions 🤷‍♂️
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers, and Happy coding!👏👏👏

## Important Notes 📝
- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**